lib/core/test/unit/Cardano/Pool/DB/Properties.hs

```diff
@@ -799,9 +798,11 @@ prop_listRegisteredPools DBLayer {..} entries =
         L.nub poolOwners /= poolOwners
 
     prop = do
+        -- TODO#1901: Is this change ok? Previously only (SlotInEpoch 0) were
+        -- generated.
```

Comment:
@jonathanknowles do you know if there was a reason for only `SlotId`s with slotInEpoch 0 were generated?
(you recently edited here)

Reply:
It seems to work 🤷‍♂️